apps/admin_app/lib/admin_app_web/controllers/order_controller.ex

```diff
@@ -43,14 +43,23 @@ defmodule AdminAppWeb.OrderController do
 
   def show(conn, %{"number" => _number} = params) do
     order = OrderContext.get_order(params)
-    order_total = OrderContext.get_total(order)
```

Comment:
This invites a change in the return type of the model and then using the `with` statements. 

{:ok, order}

{:error, msg}